### ISSUE TITLE
Is there any solution to run the app using v4 installed on server

### ISSUE THREAD

**Issue by arslannawaz** [invoiceninja/admin-portal#263]
![WhatsApp Image 2021-04-30 at 4 46 09 AM](https://user-images.githubusercontent.com/46641425/116633199-53aacb00-a972-11eb-9758-e51075470190.jpeg)

I have pulled the code from master branch, installed dependencies and made it run successfully, but it needs v5. I don't have v5 installed on my server. Is there a way so I make it work using v4? I just don't want to update from v4 to v5.

**Comment by hillelcoren** [comment]
Do you mean the mobile app? There are separate v4 and v5 versions.

**Comment by arslannawaz** [comment]
> Do you mean the mobile app? There are separate v4 and v5 versions.

Actually my current web version of ninja invoice is 4.534. I have installed app from appstore and it worked perfect with my current self host version. But I want to change some stuff in mobile app code, so I have pulled the code of mobile app from master branch and run the app in my emulator but it says that my self host version should be v5.

I don't want to upgrade my self host version of ninja invoice from v4.5 to v5. So is there a way to run the app in my emulator with self host version 4.5

Basically I just need the code of mobile app which is working perfect with self host v4.5

**Comment by hillelcoren** [comment]
The code for the v4 mobile app is on the legacy branch but I'm not sure if it will compile with the latest version of Flutter
